webpack/play.jsx

```diff
@@ -2,29 +2,56 @@ import React from "react";
 import PropTypes from "prop-types";
 import { render } from "react-dom";
 import { Provider } from "react-redux";
+
+import Acts from "./components/Acts";
+import IntermediaTable from "./components/IntermediaTable";
 import MasterVideo from "./components/MasterVideo";
-import Dance from "./components/Dance";
-import TimelineIndicator from "./components/TimelineIndicator";
 import Narrative from "./components/Narrative";
-import store from "./store";
+import TimelineIndicator from "./components/TimelineIndicator";
+import TimeMarks from "./components/TimeMarks";
+import ShodanTimeline from "./components/ShodanTimeline";
+
 import contents from "./contents";
+import store from "./store";
 
 const App = props => (
   <Provider store={store}>
-    <div className="app">
-      <Narrative narrative={props.narrative} />
-      <MasterVideo videoUrl={props.videoUrl} startTime={props.startTime} />
-      <Dance />
-      <TimelineIndicator
-        currentTime={props.currentTime}
-        duration={props.videoDuration}
-        playing={props.isPlaying}
-      />
+    <div className="video-player">
```

Comment:
Does it seem strange to have the class 'video-player' on this div as well as an internal div that continues the actual video component (see line 25)?